File: Java/Week_1/Day6IfElse.java
public class Day6IfElse {
    public static void main(String[] args) {
        
        // Basic if-else
        int age = 20;
        if (age >= 18) {
            System.out.println("You are an adult");
        } else {
            System.out.println("You are a minor");
        }
        
        // if-else if-else (multiple conditions)
        int marks = 75;
        if (marks >= 90) {
            System.out.println("Grade: A");
        } else if (marks >= 75) {
            System.out.println("Grade: B");
        } else if (marks >= 50) {
            System.out.println("Grade: C");
        } else {
            System.out.println("Grade: F");
        }
        
        // Nested if (if inside another if)
        int temperature = 30;
        boolean isRaining = false;
        
        if (temperature > 25) {
            if (!isRaining) {
                System.out.println("Hot and dry - go swimming");
            } else {
                System.out.println("Hot but raining - stay inside");
            }
        } else {
            System.out.println("Not hot enough for swimming");
        }
        
        // Combining logical operators with if-else
        int age2 = 16;
        boolean hasPermission = true;
        
        if (age2 >= 18 || hasPermission) {
            System.out.println("Allowed to enter");
        } else {
            System.out.println("Not allowed");
        }
    }
}

File: Python/Week_1/day6_if_else.py
# Day 6 - If-else Statements in Python

# Basic if-else
age = 20
if age >= 18:
    print("You are an adult")
else:
    print("You are a minor")

# if-elif-else (multiple conditions)
marks = 75
if marks >= 90:
    print("Grade: A")
elif marks >= 75:
    print("Grade: B")
elif marks >= 50:
    print("Grade: C")
else:
    print("Grade: F")

# Nested if (if inside another if)
temperature = 30
is_raining = False

if temperature > 25:
    if not is_raining:
        print("Hot and dry - go swimming")
    else:
        print("Hot but raining - stay inside")
else:
    print("Not hot enough for swimming")

# Combining logical operators with if-else
age2 = 16
has_permission = True

if age2 >= 18 or has_permission:
    print("Allowed to enter")
else:
    print("Not allowed")